keyboards/melgeek/mj63/rev1/config.h

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2020 MelGeek <melgeek001365@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
```

Comment:
You have a via keymap included in this PR, and they generally don't like this ID. Changing it within their guidelines is one of the quickest ways forward https://caniusevia.com/docs/configuring_qmk#changes-to-keyboard-directorys-configh.

Reply:
@zvecr please check it Thank you